pages/index.jsx

```diff
@@ -1,17 +1,34 @@
-import React from "react";
+import React, { useState, useEffect }  from "react";
 import Link from "next/link";
 import Head from "next/head";
+import axios from "axios";
 
 import PrimaryButton from "./components/primary-button";
 import FeatureCard from './components/feature-card'
 
-
 const Home = (props) => {
+
+const [stats, setStats] = useState([]);
+
+  let api = "https://developersdungeon.xyz/assets/json/servers.json"
+
+  useEffect(() => {
+    axios.get(api)
+    .then(res => {
+      setStats(res.data)
+    }).catch(error => console.log(error))
+  }, []);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'api'. Either include it or remove the dependency array.](https://app.codacy.com/gh/mezotv/wouldyou-web/pullRequest?prid=9892060)
```suggestion
  }, [api]);
```